### DISCUSSION TITLE
Microsoft OpenJDK is not available for Ubuntu 22.04 as a package (msopenjdk-17), contrary to what the manual says (https://docs.microsoft.com/en-us/java/openjdk/install). So my question is: Do you plan to add the package to the Microsoft repository?

### DISCUSSION THREAD

**Discussion by Lolothepro** [microsoft/openjdk#261] [Q&A] [answered]
Microsoft OpenJDK is not available for Ubuntu 22.04 as a package (msopenjdk-17), contrary to what the manual says (https://docs.microsoft.com/en-us/java/openjdk/install).
So my question is:
Do you plan to add the package to the Microsoft repository?

**Comment by d3r3kk** [comment, marked as answer]
Thank you for raising this. We are going to support LTS versions for Ubuntu as we progress.

We will add this distribution to our package server in the coming weeks.

As a workaround you can opt to use the tar.gz file (as stated in our Ubuntu installation documentation) as this method should work for you until we add publishing to the 22.04 repository.
